### PR TITLE
setup: up2: Fix error in entering BIOS instruction

### PR DESCRIPTION
**Pull request by stripes416**
Signed-off-by: Kelly Ledford <kelly.ledford@intel.com>

### CODE REVIEW

**Comment by stripes416** [comment]
Fred and I both tried it today, nothing worked but the DEL key. I found it strange as well.
> Are you sure?
> https://wiki.up-community.org/Bios_Update
> During boot, hit F7 on the usb keyboard to enter the UEFI BIOS boot menu.

**Comment by intelkevinputnam** [comment]
@stripes416 Given you're already using an all cap convention, I don't think it's necessary to add a less obvious way of doing things for folks not familiar with reStructuredText and Sphinx. Might as well stick with what works.

File: getting_started/setup/setup_up_2_board.rst [hunk 1]
```diff
@@ -28,10 +28,16 @@ will show UP-APL01 R3.6.
    
      this link needs to be updated to something accessible externally
 
-* Press F2 to enter BIOS main menu and check the BIOS version.
-* Press ENTER when prompted for password.
-* If the current BIOS version is older than 1.8, please update to 1.8
+* If the current BIOS version is older than 1.8, please update to v1.8
   before flashing v3.6.
+  .. note::
```
Comment (intelkevinputnam) [inline getting_started/setup/setup_up_2_board.rst]: Hi @stripes416,

I don't think the word "either" is necessary and the sentence "check your BIOS version press" needs a ":" before the list starts. 

There are a couple of issues here that impact rendering:

1. There needs to be a carriage return between v3.6. and .. note::
2. Bullets seem unnecessary in note section.
3. Make sure that the indentations render correctly in the final build.
4. Is the "Press ENTER" portion supposed to be in the note?

File: getting_started/setup/setup_up_2_board.rst [hunk 2]
```diff
@@ -28,10 +28,17 @@ will show UP-APL01 R3.6.
...
 
-* Press F2 to enter BIOS main menu and check the BIOS version.
-* Press ENTER when prompted for password.
-* If the current BIOS version is older than 1.8, please update to 1.8
+* If the current BIOS version is older than 1.8, please update to v1.8
   before flashing v3.6.
+
+  .. note::
+
+    To check your BIOS version press
```
Comment (intelkevinputnam) [inline getting_started/setup/setup_up_2_board.rst]: @stripes416 Line 36: Need a carriage return after this as the interpreter is getting confused about what is intended. That's why this line is bolded when click "view file."

Comment (intelkevinputnam) [reply getting_started/setup/setup_up_2_board.rst]: And no need to indent 1) and 2). This should work:

  .. note::

    To check your BIOS version press

    1) DELETE or
    2) F7 and select 'Enter Setup'

Comment (stripes416) [reply getting_started/setup/setup_up_2_board.rst]: @intelkevinputnam --- I was wondering why that was in bold!

**Comment by intelkevinputnam** [comment]
@stripes416 LGTM :-)

**Comment by lgirdwood** [comment]
@intelkevinputnam please feel free to merge unless you need further technical review where @thesofproject/maintainers can be pinged to provide the technical review.

**Comment by intelkevinputnam** [comment]
@stripes416 Changes are now live at https://thesofproject.github.io Can you do a quick check to make sure everything looks correct?

**Pull request merged**